MapboxCoreNavigationTests/TileRoutingVersionTests.swift

```diff
@@ -0,0 +1,56 @@
+@testable import MapboxCoreNavigation
+import MapboxDirections
+import XCTest
+
+class TileRoutingVersionTests: XCTestCase {
+    func testCurrentVersionReturnsExpectedValueFirstTime () {
+        // Given
+        let routeTilesVersion = RouteTilesVersion(with: DirectionsCredentials())
+        let expectedCurrentVersion = "2020_07_03-03_00_00"
+        
+        // When
+        let currentVersion = routeTilesVersion.currentVersion
+        
+        // Then
+        XCTAssertEqual(currentVersion, expectedCurrentVersion)
+    }
+    
+    func testCurrentVersionStoresValuePersistently () {
```

Comment:
Not sure what's the idea of `testCurrentVersionStoresValuePersistently` and `testCurrentVersionReadsValuePersistently`. Makes sense to revise in future.

Reply:
I’m going to remove these tests, as well as the `currentVersion` property that they would’ve exercised. I don’t see anything making use of it, and the navigator is already able to persist the version for its lifetime.